site/management.md

```diff
@@ -335,6 +348,61 @@ curl -i --header "authorization: Bearer &lt;token&gt;" http://localhost:15672/ap
 This is true for all endpoints except `GET /definitions` and `POST /definitions`. Those
 endpoints require the token to be passed in the `token` query string parameter.
 
+### Configure Scopes to request during authentication
+
+We can configure which OAuth2 **scopes** RabbitMQ should claim when redirecting the user to the authorization server by
+setting `management.oauth_scopes`.
+
+If we do not specify any scopes, when `management.enable_uaa = true`, RabbitMQ defaults to
+<pre class="lang-ini">
+management.oauth_scopes = "openid profile rabbitmq.*".
+</pre>
+
+Where `rabbitmq` is the *resource_server_id* we declared in the oauth2 plugin configuration:
+<pre class="lang-ini">
+ {rabbitmq_auth_backend_oauth2, [
+    {resource_server_id, &lt;&lt;"rabbitmq"&gt;&gt;},
+</pre>
+
+In the contrary, when `management.enable_uaa = false` or when it is not set, RabbitMQ defaults to
+<pre class="lang-ini">
+management.oauth_scopes = "openid profile"
+</pre>
+This configuration implies that your authorization server must able to
+grant additional scopes otherwise having only these 2 scopes will not grant the user access to the management ui.
+
```

Comment:
What does this mean exactly: "This configuration implies that your authorization server must able to
grant additional scopes otherwise having only these 2 scopes will not grant the user access to the management ui."

I think the above needs to be made clearer, why are we telling them about these 2 scopes at this point in the documentation if they don't give access to the management UI, surely we should be documenting the minimum scopes that do first?